challenge/src/components/chat/ChatBox.jsx

```diff
@@ -1,6 +1,6 @@
 import React, {Component} from 'react';
 import styles from './ChatBox.module.css';
-
+import Troll from '/Users/dhruvchaudhari/Desktop/Reactment/challenge/src/components/images/Troll.png';
```

Comment:
Please use relative paths instead of absolute.

Reply:
Okay, I switched it up. Thank You.